### PR TITLE
feat:Add a new theme light and dark

### PR DESCRIPTION
**Pull request by ashrafML**
The user has the ability to switch between light and dark themes using the toggle button.

### CODE REVIEW

File: config.toml.template
Comment (sa64r) [inline config.toml.template:1]: Probably do this in another PR for cleanliness given this PR is about adding theming

File: requirements.txt
Comment (sa64r) [inline requirements.txt:1]: You probably do not want to include this in the PR

File: requirement
Comment (sa64r) [inline requirement:1]: You probably do not want to include this in the PR

**Review by sa64r — COMMENTED** [review]
There's a few things that you probably want to remove from this PR that are not relevant before anyone should approve it

File: requirement
Comment (ashrafML) [reply requirement:1]: Sure thanks

File: requirements.txt
Comment (ashrafML) [reply requirements.txt:1]: Sure I will do that

**Comment by ashrafML** [comment]
Will Remove unnecessary files
